src/views/AddItem.jsx

```diff
@@ -109,57 +118,113 @@ export function AddItem({ listToken, data }) {
 
 	return (
 		<>
-			<Form onSubmit={onFormSubmit}>
-				<Form.Label htmlFor="item">Item Name:</Form.Label>
-				<InputGroup>
-					<Form.Control
-						type="text"
-						id="item"
-						name="item"
-						value={itemName}
-						onChange={(e) => setItemName(e.target.value)}
-						required={true}
-					/>
-				</InputGroup>
-				<Form.Label id="purchase-date-label">
-					How soon will you buy this again?
-				</Form.Label>
-				<fieldset>
-					<div>
-						<Form.Check
-							defaultChecked
-							aria-labelledby="purchase-date-label"
-							type="radio"
-							value="soon"
-							id="soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							value="kind-of-soon"
-							id="kind-of-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Kind of Soon"
-						/>
-					</div>
-					<div>
-						<Form.Check
-							type="radio"
-							id="not-soon"
-							value="not-soon"
-							name="radio-btn"
-							onChange={handlePurchaseDate}
-							label="Not Soon"
-						/>
-					</div>
-				</fieldset>
-				<Button type="submit">Add Item</Button>
-			</Form>
+			<div className="add-item">
+				<header className="add-item-header">
+					<h1>Add Item</h1>
+				</header>
+
+				<Container className="add-item-form">
+					<Form onSubmit={onFormSubmit}>
+						<div className="form-group">
+							<Form.Label htmlFor="item">Item Name:</Form.Label>
+							<InputGroup>
+								<Form.Control
+									type="text"
+									id="item"
+									name="item"
+									value={itemName}
+									onChange={(e) => setItemName(e.target.value)}
+									required={true}
+									placeholder="e.g., croissants"
+									className="item-input"
+								/>
+							</InputGroup>
+						</div>
+						<div className="form-group">
+							<Form.Label id="purchase-date-label">
+								How soon will you buy this again?
+							</Form.Label>
+							<fieldset>
+								<Row className="soon-row purchase-date-row">
+									<Col className="soon-radio-button">
+										<Form.Check
+											defaultChecked
+											aria-labelledby="purchase-date-label"
+											type="radio"
+											value="soon"
+											id="soon"
+											name="radio-btn"
+											onChange={handlePurchaseDate}
+											className="radio-button"
+										/>
+									</Col>
+									<Col xs={8}>
+										<Form.Label className="radio-button-label" htmlFor="soon">
+											Soon (1-7 days)
+										</Form.Label>
+									</Col>
+									<Col xs={2}>
+										<Image src="../../public/img/soon-toast.png"></Image>
```

Comment:
Might be best to `import` the image instead of directly using the image src path.

```
import soonToast from "/img/soon-toast.png";
```
Note, you don't have to include `../../public` in the path. Anything inside the `public` folder is accessible by its relative path without including the word "public."

Then in the code you can just do `<Image src={soonToast} />`

Reply:
Awesome! Will look into this and make the adjustments.